docs/content/reference/aggregated-docs.md

```
@@ -10350,6 +10350,290 @@ creating a new HostedCluster, a warning message will be displayed stating there
+This guide shows how to configure image registry mirrors for HyperShift hosted clusters.
+
+## Prerequisites
+
+- A HyperShift management cluster
+- Appropriate permissions to create HostedCluster and NodePool resources
+
+## Configure Digest-based Mirrors (IDMS)
+
+Digest-based mirrors are automatically created from `imageContentSources` in your HostedCluster:
+
+```yaml
```

markdownlint-cli2 (0.18.1)

Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)